ci-operator/step-registry/hypershift/kubevirt/baremetalds/redhat-operators/hypershift-kubevirt-baremetalds-redhat-operators-commands.sh

```diff
@@ -5,60 +5,205 @@ set -o errexit
 set -o pipefail
 set -x
 
-name="redhat-operators-$(echo $REDHAT_OPERATORS_INDEX_TAG| sed "s/[.]/-/g")"
-
-oc apply -f - <<EOF
-apiVersion: operators.coreos.com/v1alpha1
-kind: CatalogSource
-metadata:
-  annotations:
-    operatorframework.io/managed-by: marketplace-operator
-    target.workload.openshift.io/management: '{"effect": "PreferredDuringScheduling"}'
-  generation: 5
-  name: $name
-  namespace: openshift-marketplace
-spec:
-  displayName: Red Hat Operators
-  grpcPodConfig:
-    nodeSelector:
-      kubernetes.io/os: linux
-      node-role.kubernetes.io/master: ""
-    priorityClassName: system-cluster-critical
-    securityContextConfig: restricted
-    tolerations:
-    - effect: NoSchedule
-      key: node-role.kubernetes.io/master
-      operator: Exists
-    - effect: NoExecute
-      key: node.kubernetes.io/unreachable
-      operator: Exists
-      tolerationSeconds: 120
-    - effect: NoExecute
-      key: node.kubernetes.io/not-ready
-      operator: Exists
-      tolerationSeconds: 120
-  icon:
-    base64data: ""
-    mediatype: ""
-  image: registry.redhat.io/redhat/redhat-operator-index:${REDHAT_OPERATORS_INDEX_TAG}
-  priority: -100
-  publisher: Red Hat
-  sourceType: grpc
-  updateStrategy:
-    registryPoll:
-      interval: 10m
-EOF
+function mirror_ccs() {
+    echo "### Mirroring the selected operators to the internal registry"
+    source "${SHARED_DIR}/packet-conf.sh"
+
+    echo "registry.redhat.io/redhat/redhat-operator-index:${REDHAT_OPERATORS_INDEX_TAG}" > /tmp/ccs-catalog-image
+    scp "${SSHOPTS[@]}" "/tmp/ccs-catalog-image" "root@${IP}:/home/ccs-catalog-image"
+    echo "${REDHAT_OPERATORS_INDEX_TAG}" > /tmp/ccs-version
+    scp "${SSHOPTS[@]}" "/tmp/ccs-version" "root@${IP}:/home/ccs-version"
+    echo "${CCS_OPERATOR_PACKAGES}" > /tmp/ccs-packages
+    scp "${SSHOPTS[@]}" "/tmp/ccs-packages" "root@${IP}:/home/ccs-packages"
+    echo "${CCS_OPERATOR_CHANNELS}" > /tmp/ccs-channels
+    scp "${SSHOPTS[@]}" "/tmp/ccs-channels" "root@${IP}:/home/ccs-channels"
+    scp "${SSHOPTS[@]}" "${CLUSTER_PROFILE_DIR}/pull-secret" "root@${IP}:/home/pull-secret"
+
+    # shellcheck disable=SC2087
+    ssh "${SSHOPTS[@]}" "root@${IP}" bash - << 'EOF'
```

Comment:
For non-file content(REDHAT_OPERATORS_INDEX_TAG,CCS_OPERATOR_CHANNELS, etc), we can accomplish it by passing variables, while for file content(${CLUSTER_PROFILE_DIR}/pull-secret), we can still use SCP.，E.g:
https://github.com/openshift/release/blob/4618e802f1a46035bf84ac3dbc34a01b443d9f45/ci-operator/step-registry/hypershift/mce/install/disconnected/hypershift-mce-install-disconnected-commands.sh#L16

Reply:
done